application/src/main/java/com/sanction/thunder/dao/dynamodb/DynamoDbHealthCheck.java

```diff
@@ -7,6 +7,9 @@
 import java.util.Objects;
 import javax.inject.Inject;
 
+/**
+ * A Dropwizard health check for DynamoDB.
```

Comment:
So I like this summary but we should keep in mind that other people may not know about all our dependencies (in this case Dynamo). This will become especially true once we have multiple DB's working with Thunder. Dropwizard can be assumed knowledge since it's a core part of the project but Dynamo is on the other end of an abstraction layer.

Reply:
Makes sense, this summary should probably be expanded to give a brief explanation of Dynamo and how to check for its health. We can do this once we agree on the format